src/subdomains/supporting/payout/strategies/payout/__tests__/payout-bitcoinbased.strategy.spec.ts

```diff
@@ -241,8 +241,8 @@ describe('PayoutJellyfishStrategy', () => {
   });
 });
 
-class PayoutJellyfishStrategyWrapper extends JellyfishStrategy {
-  protected readonly logger = new DfxLogger(PayoutJellyfishStrategyWrapper);
+class PayoutBitcoinbasedStrategyWrapper extends BitcoinbasedStrategy {
+  protected readonly logger = new DfxLogger(PayoutBitcoinbasedStrategyWrapper);
```

Comment:
Replace `Bitcoinbased` -> `BitcoinBased` (everywhere)
In file names `bitcoinbased` -> `bitcoin-based`